usecases/mu/mpc/config.go

```diff
@@ -0,0 +1,98 @@
+	ValueConstraintsPhaseC *model.MeasurementConstraintsDataType // The constraints for the current for phase C (optional can be nil) (requires ValueSourcePhaseC to be set)
+}
+
+// MonitorVoltageConfig is the configuration for the monitor use case
+// If this config is passed via NewMPC, the use case will support voltage monitoring
+// The voltage phases will be the same as specified in MonitorPowerConfig
+type MonitorVoltageConfig struct {
+	ValueSourcePhaseA *model.MeasurementValueSourceType // The source of the values for phase A (required if the phase is supported)
+	ValueSourcePhaseB *model.MeasurementValueSourceType // The source of the values for phase B (required if the phase is supported)
+	ValueSourcePhaseC *model.MeasurementValueSourceType // The source of the values for phase C (required if the phase is supported)
+
+	ValueConstraintsPhaseA *model.MeasurementConstraintsDataType // The constraints for the voltage for phase A (optional can be nil) (requires ValueSourcePhaseA to be set)
+	ValueConstraintsPhaseB *model.MeasurementConstraintsDataType // The constraints for the voltage for phase B (optional can be nil) (requires ValueSourcePhaseB to be set)
+	ValueConstraintsPhaseC *model.MeasurementConstraintsDataType // The constraints for the voltage for phase C (optional can be nil) (requires ValueSourcePhaseC to be set)
+
+	SupportPhaseToPhase  bool                              // Set to true if the use case supports phase to phase voltage monitoring
+	ValueSourcePhaseAToB *model.MeasurementValueSourceType // The source of the values for phase A to B (required if the phases are supported and SupportPhaseToPhase is true)
+	ValueSourcePhaseBToC *model.MeasurementValueSourceType // The source of the values for phase B to C (required if the phases are supported and SupportPhaseToPhase is true)
+	ValueSourcePhaseCToA *model.MeasurementValueSourceType // The source of the values for phase C to A (required if the phases are supported and SupportPhaseToPhase is true)
+
+	ValueConstraintsPhaseAToB *model.MeasurementConstraintsDataType // The constraints for the voltage for phase A to B (optional can be nil) (requires ValueSourcePhaseAToB to be set)
+	ValueConstraintsPhaseBToC *model.MeasurementConstraintsDataType // The constraints for the voltage for phase B to C (optional can be nil) (requires ValueSourcePhaseBToC to be set)
+	ValueConstraintsPhaseCToA *model.MeasurementConstraintsDataType // The constraints for the voltage for phase C to A (optional can be nil) (requires ValueSourcePhaseCToA to be set)
+}
+
+// MonitorFrequencyConfig is the configuration for the monitor use case
+type MonitorFrequencyConfig struct {
+	ValueSource      *model.MeasurementValueSourceType     // The source of the values (required)
+	ValueConstraints *model.MeasurementConstraintsDataType // The constraints for the frequency values (optional can be nil)
+}
+
+func (c *MonitorPowerConfig) SupportsPhases(phase []string) bool {
```

Comment:
It would be great to have some tests for this method standalone as well. So line 93 is also covered.

Reply:
We added tests for this to mu/mpc/config_test.go